troubleshooting.md

```diff
@@ -0,0 +1,211 @@
+# Troubleshooting guide
+
+This guide will help you sort out the most common issues of running ITS on the PiDP-10.
+Issues covered so far:
+
+* [COMSAT is crashing on start](#comsat-is-crashing-on-start)
+
+through out the document you will see certain special characters mentioned. They are slightly different depending on your keyboard and terminal
+* `$` means the ALT or ESC key
```

Comment:
I think it is dangerous to mention ALT. The ALT key on many keyboards is a modifier key that must be held down while another key is pressed. This is not the same as ESC, which is a key intended to pressed on its own, or optionally with a modifier keys

Suggest removing ALT here as people will try to use it in place of ESC to no effect.